tests/compute/subresources/pausing.go

```diff
@@ -84,7 +85,7 @@ var _ = compute.SIGDescribe("[rfe_id:3064][crit:medium][vendor:cnv-qe@redhat.com
 			Eventually(matcher.ThisVMI(vmi), 30*time.Second, time.Second).Should(matcher.HaveConditionTrue(v1.VirtualMachineInstanceReady))
 		})
 
-		It("[test_id:3224]should not be paused with a LivenessProbe configured", func() {
+		It("[test_id:3224]should not be paused with a LivenessProbe configured", decorators.Conformance, func() {
```

Comment:
Also this is not Conformance `Expect(err).To(MatchError(ContainSubstring("Pausing VMIs with LivenessProbe is currently not supported")))`, sounds like unit test.